pkg/s3wrapper/util.go

```diff
@@ -0,0 +1,25 @@
+package s3wrapper
+
+import (
+	"net/url"
+	"os"
+)
+
+func FixEndpointURL(endpoint string) (string, error) {
+	_, err := url.ParseRequestURI(endpoint)
+	if err == nil {
+		return endpoint, nil
+	}
+
+	prefix := "http://"
+	if os.Getenv("S3_USE_SSL") == "true" {
```

Comment:
I think it will be cleaner to have it as part of options and give it as a parameter because we don't use `os.Getenv` anywhere else in the code and all the configuration is set in `Options`, but it's not a must.

Reply:
It seemed like overkill to me as this is the only place it is being used.